docs/INTEGRATION.md

```diff
@@ -29,6 +29,10 @@ Then add the CI step on codeship-steps.yml with the path of the dockerfile
     - service: hadolint
       command: hadolint /test/Dockerfile
 ```
+### Mega-Linter
+
+[Mega-Linter](https://nvuillam.github.io/mega-linter/) aggregates **70** linters, including [hadolint](
```

Comment:
IMHO this text is short enough that using bold text is unnecessary.

Reply:
updated